auspice/server/getDataset.js

```diff
@@ -40,6 +40,20 @@ const requestMainDataset = async (res, req, fetchUrls) => {
   res.send(datasetJson);
 };
 
+/**
+ * "inrb-drc" was the first of the Nextstrain groups. Groups now live at
+ * `nextstrain.org/groups`, but we want to support old URLs for INRB DRC by
+ * redirecting requests from "/inrb-drc" to "/groups/inrb-drc".
+ *
+ * @param {Response} res
+ * @param {Object} query
+ */
+const redirectInrbDrc = (res, query) => {
+  if (query.prefix.startsWith('/inrb-drc')) {
+    res.redirect("getDataset?prefix=/groups" + query.prefix);
```

Comment:
Nice work fixing this bug @kairstenfay. As a heads up for future nextstrain work, a solution such as this would break the sourcing of any sidecar files, which are defined by additional parts of the query (see https://nextstrain.github.io/auspice/server/api#charon-getdataset). As this group doesn't use any sidecar files (e.g. tip-frequencies) I would expect this solution to be perfectly acceptable.

Reply:
The `query.prefix` value needs to be escaped before being added to the URL. While you can do this with `encodeURIComponent()`, using `"getDataset?" + queryString.stringify(query)` will both do the escaping and address the sidecar issue James mentioned (since it would include `query.type` too). Note that `query` originally came from `queryString.parse(…)`; `stringify` is the inverse operation.